src/components/Main/Main.js

```diff
@@ -48,12 +49,18 @@ const Main = props => {
 
   return (
     <Styled.Content>
+      <style>
```

Comment:
yooo I didn't even know you could toss this into a component like this, that's kinda sick